globalization/localization.md

```diff
@@ -24,11 +24,11 @@ The Telerik UI for Blazor components use a set of keys that a localization servi
 You can find the list of keys in the following places:
 
 * In the [API reference](../api/Telerik.Blazor.Resources.Messages.html)
-* By inspecting the `Resources/Messages.resx` file in the offline Demos installation (under a path like `C:\Program Files (x86)\Progress\Telerik UI for Blazor <YOUR VERSION>\demos\TelerikBlazorDemos\Resources`). There you can also see the default values.
+* By inspecting the `Resources/TelerikMessages.resx` file in the offline Demos installation (under a path like `C:\Program Files (x86)\Progress\Telerik UI for Blazor <YOUR VERSION>\demos\TelerikBlazorDemos\Resources`). There you can also see the default values.
```

Comment:
The bullet list is introduced and a list of "places", so we can't start an item with "by inspecting".

```suggestion
* In the `Resources/TelerikMessages.resx` file in the offline Demos installation (under a path like `C:\Program Files (x86)\Progress\Telerik UI for Blazor <YOUR VERSION>\demos\TelerikBlazorDemos\Resources`). There you can also see the default English values.
```